infra/feast-operator/test/api/featurestore_types_test.go

```diff
@@ -67,6 +67,41 @@ func onlineStoreWithRelativePathForEphemeral(featureStore *feastdevv1alpha1.Feat
 	return copy
 }
 
+func onlineStoreWithS3BucketForPvc(featureStore *feastdevv1alpha1.FeatureStore) *feastdevv1alpha1.FeatureStore {
+	copy := featureStore.DeepCopy()
+	copy.Spec.Services = &feastdevv1alpha1.FeatureStoreServices{
+		OnlineStore: &feastdevv1alpha1.OnlineStore{
+			Persistence: &feastdevv1alpha1.OnlineStorePersistence{
+				FilePersistence: &feastdevv1alpha1.OnlineStoreFilePersistence{
+					Path: "s3://bucket/online_store.db",
+					PvcConfig: &feastdevv1alpha1.PvcConfig{
+						Create:    &feastdevv1alpha1.PvcCreate{},
+						MountPath: "/data/online",
+					},
+				},
+			},
+		},
+	}
+	return copy
+}
+func onlineStoreWithGsBucketForPvc(featureStore *feastdevv1alpha1.FeatureStore) *feastdevv1alpha1.FeatureStore {
+	copy := featureStore.DeepCopy()
+	copy.Spec.Services = &feastdevv1alpha1.FeatureStoreServices{
+		OnlineStore: &feastdevv1alpha1.OnlineStore{
+			Persistence: &feastdevv1alpha1.OnlineStorePersistence{
+				FilePersistence: &feastdevv1alpha1.OnlineStoreFilePersistence{
+					Path: "gs://bucket/online_store.db",
```

Comment:
above method `onlineStoreWithS3BucketForPvc` this one appears to be the same except the difference in the path. can we pass as method param?